app/main.py

```diff
@@ -1,13 +1,16 @@
+
+
+
 def format_linter_error(error: dict) -> dict:
-    # write your code here
+    return {"line": error["line_number"], "column": error["column_number"], "message": error["text"], "name": error["code"], "source": "flake8"}
```

Comment:
The 'pass' statement on line 6 is unnecessary because the function already has a return statement. You can remove it.